packages/ui/src/lib/Modal.svelte

```diff
@@ -1,4 +1,8 @@
-<script lang="ts">
+<script lang="ts" module>
+	type T = any | unknown | undefined;
+</script>
+
+<script lang="ts" generics="T extends undefined | any = any">
```

Comment:
Not super important right now, but see the `as any` below. Would be great to have a path for making the usages of this component more type safe. To find where the generics is currently used, look for the `defaultItem` parameter.